modules/client-configuration/pages/clients-rh-rhui.adoc

```diff
@@ -0,0 +1,424 @@
+[[clients-rh-rhui]]
+= Registering {rhel} Clients with RHUI
+
+If you are running {rhel} clients directly, rather than using {sleses}, you need to use Red Hat sources to retrieve and update packages.
+This section contains information about using the {redhat} update infrastructure (RHUI) to register traditional and Salt clients running {rhel} operating systems.
+
+For information about using {redhat} content delivery network (CDN) instead, see xref:client-configuration:clients-rh-cdn.adoc[].
+
+
+[IMPORTANT]
+====
+{rhel} clients are based on {redhat} and are unrelated to {sleses}, RES, {redhat}, or {sles}.
+You are responsible for arranging access to {redhat} base media repositories and {rhela} installation media, as well as connecting {productname} Server to the {redhat} content delivery network.
+You must obtain support from {redhat} for all your {rhela} systems.
+If you do not do this, you might be violating your terms with {redhat}.
+====
+
+
+[IMPORTANT]
+====
+Traditional clients are available on {rhel}{nbsp}6 and 7 only.
+{rhel}{nbsp}8 clients are supported as Salt clients.
+====
+
+
+
+== Import Entitlements and Certificates
+
+{redhat} clients require a {redhat} certificate authority (CA) and entitlement certificate, and an entitlement key.
+
+Entitlement certificates are embedded with expiration dates, which match the length of the support subscription.
+To avoid disruption, you will need to repeat this process at the end of every support subscription period.
```

Comment:
Maybe. Or maybe not. I'm not sure this is valid for RHUI.

Could you ask for second advice to someone "who knows better" please? Maybe @meaksh .

Reply:
Sure. @meaksh ?

Reply:
This seems to be a general note about how susbscriptions works on the RHEL world, particularly helpful when dealing directly with the CDN, where the entitlements you are related to your own subscription. So you need to take care of update the entitlement every time your subscription expires.

But for RHUI, the entitlement are not yours but owned by the cloud provider, so you don't really know when they expired. Maybe we should avoid this particular text on the RHUI guide.